src/catalog/src/builtin.rs

```diff
@@ -2126,6 +2126,7 @@ pub static MZ_TABLES: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
         )
         .with_column("create_sql", ScalarType::String.nullable(true))
         .with_column("redacted_create_sql", ScalarType::String.nullable(true))
+        .with_column("insert_only", ScalarType::Bool.nullable(false))
```

Comment:
This is insta-stable, which is maybe a problem if we need to rip this out while it's in private preview.

Reply:
+1, making this column insta stable is a little scary, let's chat in Slack